portal/models/questionnaire.py

```diff
@@ -0,0 +1,47 @@
+"""Questionnaire module"""
+from sqlalchemy.dialects.postgresql import JSONB
+
+from ..database import db
+
+
+class Questionnaire(db.Model):
+    __tablename__ = 'questionnaires'
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.Text, nullable=False, unique=True)
```

Comment:
Why not make the title (AKA `instrument_id`) the primary key? 

No strong feelings either way, will probably continue to use `instrument_id` in the developer-facing parts of the API